js/test/units/testunitfmt_en_GB.js

```diff
@@ -340,5 +340,209 @@ module.exports.testunitfmt_en_GB = {
+    },
+    testUnitFormatWithUsagePersonWeight_Long_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "kg",
+            amount: 58
+        });
+
+        var uf = new UnitFmt({
+            usage: "personWeight",
+            measurementSystem: "imperial",
+            length: "long",
+            locale: "en-GB"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "128 pounds");
+        test.done();
+    },
+    testUnitFormatWithUsagePersonHeight_Short_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "cm",
+            amount: 178
+        });
+
+        var uf = new UnitFmt({
+            usage: "personHeight",
+            measurementSystem: "imperial",
+            length: "short",
+            locale: "en-GB"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "5 ft, 10 in");
+        test.done();
+    },
+    testUnitFormatWithUsagePersonHeight_Long_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "cm",
+            amount: 178
+        });
+
+        var uf = new UnitFmt({
+            usage: "personHeight",
+            measurementSystem: "imperial",
+            length: "long",
+            locale: "en-GB"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "5 feet, 10 inches");
+        test.done();
+    },
+    testUnitFormatWithUsageBabyHeight_Short_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "cm",
+            amount: 40
+        });
+
+        var uf = new UnitFmt({
+            usage: "babyHeight",
+            measurementSystem: "imperial",
+            length: "short",
+            locale: "en-GB",
+            style: "numeric"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "15.7 in");
+        test.done();
+    },
+    testUnitFormatWithUsageBabyHeight_Long_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "cm",
+            amount: 40
+        });
+
+        var uf = new UnitFmt({
+            usage: "babyHeight",
+            measurementSystem: "imperial",
+            length: "long",
+            locale: "en-GB",
+            style: "numeric"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "15.7 inches");
+        test.done();
+    },
+    testUnitFormatWithUsageBabyWeight_Short_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "kg",
+            amount: 14
+        });
+
+        var uf = new UnitFmt({
+            usage: "babyWeight",
+            measurementSystem: "imperial",
+            length: "short",
+            locale: "en-GB",
+            style: "numeric"
+        });
+        var str = uf.format(m1);
+        test.equal(str, "30 lb, 14 oz");
```

Comment:
A 30 pound baby?!?!? My God, that mother probably has permanent damage after that birth.